src/context/index.ts

```diff
@@ -25,7 +26,10 @@ const nonPrimitiveProps: (keyof Config)[] = [
   'INCLUDED_PROPS',
 ];
 
-export const setupContext = async (config: Config): Promise<DirectoryContext | YAMLContext> => {
+export const setupContext = async (
+  config: Config,
+  command: 'import' | 'export'
```

Comment:
It was necessary for me to introduce this command argument to exclude the prerequisite checks if an import was being run.